{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/apps/common/formatters.py

```diff
@@ -0,0 +1,14 @@
+from pythonjsonlogger import jsonlogger
+
+
+class DjangoRequestJsonFormatter(jsonlogger.JsonFormatter):
+    def add_fields(self, log_record, record, message_dict):
+        if getattr(record, 'request', None):
+            if getattr(record.request, 'method', None):
```

Comment:
@Struchu I know this was per your request, but I'd be more surgical about it. Maybe check if content-type is `application/json` first? 

What I want to avoid is logging all requests, including oversized POST bodies like file attachments. If you forget that it happens, logs will grow alongside persistent storage and go out of control easily.